frontend/src/features/workspace/components/AdminSwitchEnvMessage.tsx

```diff
@@ -1,23 +1,26 @@
 // TODO #4279: Remove after React rollout is complete
-import { Text } from '@chakra-ui/react'
+import { Text, useDisclosure } from '@chakra-ui/react'
 
 import Button from '~components/Button'
 import InlineMessage from '~components/InlineMessage'
 
-import { useEnvMutations } from '~features/env/mutations'
+import { SwitchEnvFeedbackModal } from '~features/env/SwitchEnvFeedbackModal'
 
 export const AdminSwitchEnvMessage = (): JSX.Element => {
-  const { adminSwitchEnvMutation } = useEnvMutations()
+  const { isOpen, onOpen, onClose } = useDisclosure()
 
   return (
-    <InlineMessage>
-      <Text>
-        Welcome to the new FormSG! You can still{' '}
-        <Button variant="link" onClick={() => adminSwitchEnvMutation.mutate}>
-          <Text as="u">switch to the original one,</Text>
-        </Button>{' '}
-        which is available until 28 May 2022.
-      </Text>
-    </InlineMessage>
+    <>
+      <InlineMessage>
+        <Text>
+          Welcome to the new FormSG! You can still
+          <Button variant="link" onClick={onOpen}>
+            <Text as="u">switch to the original one,</Text>
+          </Button>
+          which is available until 28 May 2022.
```

Comment:
> until 28 May 2022

LOL, probably need to change that to, hmmm, let's say `15 September 2022` for now? 😅

Reply:
hahahhaha I've changed the date to an env var in case we need to change it again in the future 😬